css/emailverification.css

```diff
@@ -0,0 +1,41 @@
+
+body{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background: rgba(196, 196, 196, 0.17);  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5674265)